src/SFA.DAS.AssessorService.Application.Api.External/Helpers/CertificateHelpers.cs

```diff
@@ -17,6 +19,10 @@ public static bool IsDraftCertificateDeemedAsReady(Certificate certificate)
             {
                 return false;
             }
+            else if (certificate.CertificateData.LearningDetails.Version is null)
```

Comment:
Good changes to make sure version is set along with option if applicable. Just a shame we don't tell them what's wrong with a certificate for submission. But that's beyond our scope